src/dc_msg.c

```diff
@@ -1093,6 +1093,34 @@ void dc_msg_set_text(dc_msg_t* msg, const char* text)
 }
 
 
+/**
+ * Set if the message is sent out encrypted or unencrypted.
+ * This overwrites the heuristic which automatically desides if a message is encrypted or
```

Comment:
```suggestion
 * This overwrites the heuristic which automatically decides if a message is encrypted or
```

Reply:
Your right, of cause. Pushed commit to fix this.